package.json

```diff
@@ -33,7 +33,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.11.2",
     "@fortawesome/react-fontawesome": "^0.1.7",
     "@storybook/addon-knobs": "^5.3.17",
-    "@storybook/react": "^5.3.17",
+    "@storybook/react": "^6.0.21",
```

Comment:
Remember to take responsibility of the commits! **Always** check files changed to make sure you have added all the file changes!

Please omit the changes in the package.json

Reply:
hi Benjamin 
my branch is clean and nothing to commit. I will keep in mind to put information about my PR in the story